examples/CMakeLists.txt

```diff
@@ -1,2 +1,10 @@
 add_executable(Example example.cpp)
-target_link_libraries(Example PUBLIC hwinfo::HWinfo)
\ No newline at end of file
+
+include(FetchContent)
+FetchContent_Declare(
+  fmt
+  GIT_REPOSITORY https://github.com/fmtlib/fmt
+  GIT_TAG e69e5f977d458f2650bb346dadf2ad30c5320281) # 10.2.1
```

Comment:
Can we explicitly state a release version (version tag) here instead?

Reply:
Fixed in [`a83c64d` (#95)](https://github.com/lfreist/hwinfo/pull/95/commits/a83c64d4958d41276dde16e09c92ddecb11069c8)